### DISCUSSION TITLE
Maximum call stack size exceeded

### DISCUSSION THREAD

**Discussion by AntDavi** [vercel/community#5495] [Help] [answered]
### Summary

I had this error while I deployed a website using nextJs, what leaves me confused is that the previous commit that is working does not have such significant differences, they are just removing some dependencies that are not being used and what is still being removed But my problem is that when restoring the repository to the committee that works on github and locally, making some changes such as adding another word in a text, the error still persists. When I build locally it passes without any further problems.

### Steps to Reproduce

![Captura de tela 2024-01-16 135046](https://github.com/vercel/community/assets/69051403/e08db448-b198-4973-a16f-80fe1a1fb62a)
[
![Captura de tela 2024-01-16 135226](https://github.com/vercel/community/assets/69051403/5f99024a-491e-4ae2-8967-082618d2f348)
](url)

**Comment by sattwyk** [comment]
I’m also getting the same error when I deploy it on Vercel, but it builds successfully locally.

![image](https://github.com/vercel/community/assets/79150484/03e24d68-4bee-4599-9f46-c867dac82e33)

![image](https://github.com/vercel/community/assets/79150484/ef2d9211-d6ec-4c9f-89fb-129008027e4a)

**Reply by sattwyk** [reply]
I got it working finally downgraded sharp to 0.33.1 
thanks to https://github.com/vercel/next.js/discussions/58372#discussioncomment-8129882

**Comment by leerob** [comment, marked as answer]
This was fixed in `v14.0.5-canary.60`: https://github.com/vercel/next.js/pull/60740
